### PR TITLE
Added prep_id as an optional parameter

### PR DESCRIPTION
**Pull request by charles-cowart**
Added prep_id as an optional parameter to
.../study/description/<study_id>?prep_id=<prep_id>.

This will bring up the prep_id workflow page instead of summary information.

### CODE REVIEW

**Comment by charles-cowart** [comment]
@antgonza ready for review. We can remove the --no-check-certificate line in GitHub workflows whenever it's appropriate.

File: .github/workflows/qiita-ci.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ jobs:
       - name: Install plugins
         shell: bash -l {0}
         run: |
-          wget https://data.qiime2.org/distro/core/qiime2-2022.11-py38-linux-conda.yml
+          wget --no-check-certificate https://data.qiime2.org/distro/core/qiime2-2022.11-py38-linux-conda.yml
```
Comment (antgonza) [inline .github/workflows/qiita-ci.yml:111]: Please rm `--no-check-certificate`

**Review by antgonza — CHANGES_REQUESTED** [review]
Works great, thank you! A couple of things:
1. if prep-id is not an integer, the code crashes :: expected behavior: do not change frame and display JS error like: "[prep-id] is not a valid preparation for this study".
2. if the prep-id doesn't exist in that study, the window hangs :: expected behavior: similar to point 1.

File: .github/workflows/qiita-ci.yml (hunk 1 above)
Comment (charles-cowart) [reply .github/workflows/qiita-ci.yml]: done!

**Comment by charles-cowart** [comment]
WIP: Unit tests for the new error-checking need to be added, and a change needs to be made to have the error-page only show up in the main frame, but this gets the functionality we need.

**Comment by antgonza** [comment]
Works as expected and code looks fine. Now, I originally thought a JS (JavaScript) error message should have been enough (see my original review) to let the user know that the prep-id doesn't exist and thought that full HTTPError might be to harsh; however, as this is already done with HTTPError, I think is fine to leave as is and if we get a user feedback we can change to a more simpler JS error + in theory these links should be computer generated.

**Review by antgonza — APPROVED** [review]

**Pull request merged**